test/scripts/e2e-kind.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -ex
+
+SHARD=$1
+
+pushd $GOPATH/src/k8s.io/kubernetes/
+export KUBERNETES_CONFORMANCE_TEST=y
+export KUBECONFIG=${HOME}/admin.conf
+export MASTER_NAME=${KIND_CLUSTER_NAME}-control-plane
+export NODE_NAMES=${MASTER_NAME}
+
+case "$SHARD" in
+	e2e-kind-ovn-shard-n)
+		# all tests that don't have P as their sixth letter after the N
+        kubetest --provider=local --deployment=kind --kind-cluster-name=${KIND_CLUSTER_NAME} --test --test_args='--ginkgo.focus=\[sig-network\]\s[Nn](.{6}[^Pp].*|.{0,6}$) --disable-log-dump=false --ginkgo.skip=Networking\sIPerf\sIPv[46]|\[Feature:PerformanceDNS\]|\[Feature:IPv6DualStackAlphaFeature\]|NetworkPolicy\sbetween\sserver\sand\sclient.+(ingress\saccess|multiple\segress\spolicies|allow\segress\saccess)|\[Feature:NoSNAT\]|Services.+(ESIPP|cleanup\sfinalizer|session\saffinity)|\[Feature:Networking-IPv6\]|\[Feature:Federation\]|configMap\snameserver|ClusterDns\s\[Feature:Example\]|(Namespace|Pod)Selector\s\[Feature:NetworkPolicy\]|kube-proxy|should\sset\sTCP\sCLOSE_WAIT\stimeout'
```

Comment:
can we make the skipped tests a single variable like before?

Reply:
happy to move it to a variable but confused by what you mean by `like before`.
it was written long form in the `yml` file in the master branch when i started work on this...

Reply:
the skipped tests were rendered by ruby from a single source:
https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflow-templates/render.rb#L13

Reply:
oh man i can't believe i missed that. it must be ruby blindness :gem: